src/main/java/uk/gov/hmcts/probate/model/ccd/raw/response/ResponseCaseData.java

```diff
@@ -323,6 +323,7 @@
     private List<CollectionMember<ParagraphDetail>> paragraphDetails = new ArrayList<>();
     private String bulkScanCaseReference;
     private String grantDelayedNotificationIdentified;
-    private final List<CollectionMember<DeathRecord>> deathRecords;
-
+    private List<CollectionMember<DeathRecord>> deathRecords;
+    private Integer deathRecordSystemNumber;
```

Comment:
same , shouldnt these both new vars be final ?

Reply:
These need to be non-final to allow the case data to be updated